crhelper/resource_helper.py

```diff
@@ -27,7 +28,7 @@
 
 class CfnResource(object):
 
-    def __init__(self, json_logging=False, log_level='DEBUG', boto_level='ERROR', polling_interval=2, sleep_on_delete=120, ssl_verify=None):
+    def __init__(self, json_logging=False, log_level='DEBUG', boto_level='ERROR', polling_interval=2, sleep_on_delete=120, ssl_verify=None, botocore_config={}):
```

Comment:
Should not use mutable types as defaults, rather use `None` as the default https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments